### PR TITLE
zenity: add recommended webkitgtk and option libnotify

### PR DESCRIPTION
**Pull request by s172262**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

Add: option libnotify.

$ ./configure --help

> `configure' configures Zenity 3.20.0 to adapt to many kinds of systems. 
> ...
> --enable-libnotify      Enable libnotify support

$ otool -L /usr/local/Cellar/zenity/3.20.0/bin/zenity |grep -v : |sed -e "s/(.*$//"

>   /usr/local/Cellar/gtk+3/3.18.9/lib/libgtk-3.0.dylib 
>     /usr/local/Cellar/gtk+3/3.18.9/lib/libgdk-3.0.dylib 
>     /System/Library/Frameworks/Cocoa.framework/Versions/A/Cocoa 
>     /usr/local/Cellar/pango/1.38.1/lib/libpangocairo-1.0.0.dylib 
>     /usr/local/Cellar/pango/1.38.1/lib/libpango-1.0.0.dylib 
>     /usr/local/lib/libatk-1.0.0.dylib 
>     /usr/local/lib/libcairo-gobject.2.dylib 
>     /usr/local/lib/libcairo.2.dylib 
>     /usr/local/opt/gdk-pixbuf/lib/libgdk_pixbuf-2.0.0.dylib 
>     /usr/local/lib/libgio-2.0.0.dylib 
>     /usr/local/lib/libgobject-2.0.0.dylib 
>     /usr/local/lib/libglib-2.0.0.dylib 
>     /usr/local/lib/libintl.8.dylib 
>     /usr/local/Cellar/libnotify/0.7.6/lib/libnotify.4.dylib 
>     /usr/lib/libSystem.B.dylib

### CODE REVIEW

**Comment by UniqMartin** [comment]
Note that there's already an open pull request that will update this formula: #267. As you can read there, Gnome 3.20 doesn't seem to be ready yet, but we still want to update those formulae together instead of selectively updating individual formulae.

I'm afraid this PR will have to wait until the mentioned PR is merged. Or maybe you can decouple the `libnotify` stuff from the version bump, if `libnotify` support is already present in the older version?

**Comment by s172262** [comment]
$ ./configure --help
> `configure' configures Zenity 3.18.1 to adapt to many kinds of systems.
>...
> --enable-libnotify      Enable libnotify support
  --enable-webkitgtk      Enable webkit support

So I close this and open new PR of 3.18.1 added two options.

**Comment by s172262** [comment]
$ brew uses zenity
is nothing!

File: Formula/zenity.rb [hunk 1]
```diff
@@ -17,6 +18,8 @@ class Zenity < Formula
   depends_on "gtk+3"
   depends_on "gnome-doc-utils"
   depends_on "scrollkeeper"
+  depends_on "webkitgtk" => :recommended
```
Comment (DomT4) [inline Formula/zenity.rb]: I'd _really_ like to avoid making this recommended if possible. It's a frankly massive dependency.

Comment (s172262) [inline Formula/zenity.rb]: This is copy of #267. And `winetricks` can work without `webkitgtk`.

Comment (dunn) [inline Formula/zenity.rb]: Let's make it `:optional`.

**Comment by DomT4** [comment]
Will happily review a fresh PR but this one seems to have gone dead, so closing. Thank you nonetheless for the submission to Homebrew ❤️

**Pull request closed without merging**